### PR TITLE
Remove broad exception for news. Remove duplicate log

### PR DESCRIPTION
**Pull request by fernandog**
~~TODO: Need to update local file every release~~

~~About news, we show a link for the news when we failed to get it from github~~

### CODE REVIEW

**Comment by p0psicles** [comment]
Do we really want to keep this in the repo? Shouldn't we just cache in db?

**Comment by fernandog** [comment]
@p0psicles used to be in local file. We moved to the repo because the file is always being updated right so user could get file soon instead of waiting for new releases. Righ @duramato ?

**Comment by p0psicles** [comment]
But then using the cache db would be preferred imo. Then you have best of both worlds.

**Comment by fernandog** [comment]
It's already in the cache DB. We get the file and import to network_timezones table

**Comment by p0psicles** [comment]
Ow. Then what's the issue? Why should we keep a file next to it?

**Comment by fernandog** [comment]
@p0psicles the issue is failing to connect to cdn to get the updates. And bigger issue is in new installs when cache.db doesn't exist

**Comment by fernandog** [comment]
@p0psicles Did you read the issue?

**Comment by fernandog** [comment]
It's also a fix for this:
https://github.com/pymedusa/Medusa/issues/3462

**Comment by labrys** [comment]
Posted my opinion on slack.

File: medusa/version_checker.py [hunk 1]
```diff
@@ -306,15 +306,12 @@ def check_for_new_news(self, force=False):
...
-            news = self.session.get(app.NEWS_URL).text
-        except Exception:
-            log.warning(u'check_for_new_news: Could not load news from repo.')
-            news = ''
-
-        if not news:
-            return ''
+        log.debug(u'Checking GitHub for latest news.')
+        response = self.session.get(app.NEWS_URL)
+        news = response.text
```
Comment (p0psicles) [inline medusa/version_checker.py]: If response is None, youl get an exception on the response.text

Comment (fernandog) [reply medusa/version_checker.py]: Will change thanks

**Comment by fernandog** [comment]
@p0psicles sorry, I did another commit and it dismissed your approve. I done now

**Pull request merged**